plugin/vmux_send_cmd

```diff
@@ -1,3 +1,4 @@
+#!/usr/bin/env bash
```

Comment:
could you use `sh` instead of bash ?
I thing it would be more compliant with other shells.

Reply:
Sorry for conflating the two pull requests, that was a mistake. The shebang commit doesn't really belong here ­— reverted. I can squash the revert if you'd like.